README.md

```diff
@@ -49,6 +54,12 @@ Any time you update the Experience Builder module or modify its front-end code,
 ddev ui-build
 ```
 
+When developing the React app, enable the `xb_vite` module (`ddev drush en xb_vite -y`), make sure you use the HTTPS URL of your DDEV project, then run:
```

Comment:
We can eliminate this manual step by enabling the `xb_vite` module automatically here: https://github.com/TravisCarden/ddev-drupal-xb-dev/blob/v0.0.16/commands/web/xb-site-install#L14-L15

Reply:
On my machine, it takes roughly a second to install or uninstall the `xb_vite` module via Drush, which seems insignificant. It makes sense to me to go ahead and build it right into the DDEV commands. If you do, be aware that Drush will cause a failure if you try to uninstall a module that isn't currently installed. I suggest doing something like this to prevent needless failure and suppress error output:

```
drush pm:uninstall xb_vite -y > /dev/null 2>&1 || :
```

And I would do the same when installing the module, just for consistency:


```
drush pm:install xb_vite -y > /dev/null 2>&1 || :
```

Reply:
I was hesitant about this at first — I thought as a developer working on my Vite config, I would like to have control over when that module gets installed and uninstalled. I realize I was wrong: as you point out, the time added to executing the commands is negligible, and this approach is a huge improvement in DX. I went ahead and made the changes in 65d9af82e3c0a40109e40803d9cdb80f12030bde. Playing with the updated commands as I was testing them felt really pleasant. 😊 Thank you!

Reply:
_Nice._ You're welcome! 😊